.github/workflows/docs.yml

```diff
@@ -46,4 +46,4 @@ jobs:
       - name: Sync docs to ReadMe
         uses: readmeio/rdme@v10
         with:
-          rdme: docs upload ./documentation/legacy/ --key=${{ secrets.README_DEVELOPERS_API_KEY }} --version=${{ vars.README_DEVELOPERS_MAIN_VERSION }} --skip-validation
```

Comment:
I assume the `--skip-validation` flag was due to our bidi sync, so removing now that we've decoupled that behavior. Lmk if that's an incorrect assumption!

Reply:
ah yep great catch, you're correct!